synapse/tests/files/rstorm/testsvc.py

```diff
@@ -0,0 +1,36 @@
+import synapse.lib.cell as s_cell
```

Comment:
Is there another testsvc which can be used? Just thinking because we're adding in yet another demonstrative testsvc implementation in python.

Reply:
I specifically wanted to test the conf values (with required values), and couldn't find any existing one that fit the bill.  Updating the existing test services would require touching a bunch of tests which didn't seem worth it.  Open to suggesetions though.

Reply:
This is fine.  Stand-alone tests tend to be more resilient anyway.